llama_index/llms/loading.py

```diff
@@ -0,0 +1,41 @@
+from llama_index.llms.base import LLM
+from llama_index.llms.custom import CustomLLM
+from llama_index.llms.huggingface import HuggingFaceLLM
+from llama_index.llms.langchain import LangChainLLM
+from llama_index.llms.llama_cpp import LlamaCPP
+from llama_index.llms.mock import MockLLM
+from llama_index.llms.openai import OpenAI
+from llama_index.llms.palm import PaLM
+from llama_index.llms.predibase import PredibaseLLM
+from llama_index.llms.replicate import Replicate
+from llama_index.llms.xinference import Xinference
+
+
+def load_llm(data: dict) -> LLM:
+    """Load LLM by name."""
+    llm_name = data.get("class_name", None)
+    if llm_name is None:
+        raise ValueError("LLM loading requires a class_name")
+
+    if llm_name == MockLLM.__name__:
```

Comment:
nit: can simplify this logic to just a list of recognized LLM classes?

Reply:
Done!